src/run.jl

```diff
@@ -8,22 +8,22 @@ function with_mounted_shards(f, runner)
 end
 
 """
-    run_sandboxed_julia(args=``; ver::VersionNumber, do_obtain=true, kwargs...)
+    run_sandboxed_julia(args=``; ver, do_obtain=true, kwargs...)
```

Comment:
I added the argument type because I remember getting some weirdness before when I passed in a string. Don't remember what it was though.

Reply:
OK, I'll have a closer look. I changed it to be able to pass `latest` into it, or generally use non-VersionNumber identifiers in Versions.toml